test/mailers/passwordless/mailer_test.rb

```diff
@@ -12,5 +12,9 @@ class Passwordless::MailerTest < ActionMailer::TestCase
     assert_match "Signing in ✨", email.subject
     assert_match /sign in: hello\n/, email.body.to_s
     assert_match %r{/sign_in/#{session.identifier}/hello}, email.body.to_s
+
+    session = Passwordless::Session.create!(authenticatable: user)
+    email = Passwordless::Mailer.sign_in(session)
+    assert_equal [user.email], email.to
```

Comment:
without [these changes](https://github.com/mikker/passwordless/pull/179/files#diff-6877b38c16e44ee59f549c74a73d881b521c96b6b536a0e52040de22fe1941daR20), error can be reproduced like this:

```
    session = Passwordless::Session.create!(authenticatable: user)
    email = Passwordless::Mailer.sign_in(session)
    assert_raises(ActionController::UrlGenerationError) do
      email.to
    end
```